tfx/experimental/pipeline_testing/pipeline_recorder_utils.py

```diff
@@ -39,26 +39,35 @@ def _get_paths(metadata_connection: metadata.Metadata, execution_ids: List[int],
 
   Args:
     metadata_connection: Instance of metadata.Metadata for I/O to MLMD.
-    execution_ids: List of execution ids of a pipeline run.
+    executions: List of executions of a pipeline run.
     output_dir: Directory path where the pipeline outputs should be recorded.
 
   Yields:
     Iterable over tuples of source uri and destination uri.
   """
-  events = metadata_connection.store.get_events_by_execution_ids(execution_ids)
-  output_events = [
-      x for x in events if x.type == metadata_store_pb2.Event.OUTPUT
-  ]
-  unique_artifact_ids = list({x.artifact_id for x in output_events})
-
-  for artifact in metadata_connection.store.get_artifacts_by_id(
-      unique_artifact_ids):
-    src_uri = artifact.uri
-    artifact_properties = artifact.custom_properties
-    component_id = artifact_properties['producer_component'].string_value
-    name = artifact_properties['name'].string_value
-    dest_uri = os.path.join(output_dir, component_id, name)
-    yield (src_uri, dest_uri)
+  for execution in executions:
+    component_id = execution.properties[
+        metadata._EXECUTION_TYPE_KEY_COMPONENT_ID].string_value  # pylint: disable=protected-access
+    if component_id == 'ResolverNode.latest_blessed_model_resolver':
```

Comment:
Maybe we should ignore all ResolverNode, rather than just `latest_blessed_model_resolver`. Out of curiosity, what is the reason to ignore Resolver nodes?